scripts/Tools/Makefile

```diff
@@ -30,20 +30,6 @@ CP    := cp
 exec_se: $(EXEC_SE)  Depends
 complib: $(COMPLIB)  Depends
 
-
-CIME_MODEL ?= $(shell $(CASEROOT)/xmlquery --caseroot $(CASEROOT) MODEL --value)
```

Comment:
This is worth a close look. I don't know who added these, but I don't think we want xmlquery calls in the Makefile since they are slow and we already have a mechanism (environment) for passing values to Make from build.py.

Reply:
I put them in and in general they are not used because the variable is already defined.  Was there a performance issue that caused you to remove them?
?= says only do this if the left hand side is not defined.